packages/adminrouter/extra/src/README.md

```diff
@@ -264,28 +264,174 @@ it, but it does not seem to be justified. For example:
  * `lib/auth/open.lua` uses the `.init()` pattern in order to achive different
    behaviour depending on the flavour.
 
-## Service Endpoints
-
-Admin Router allows Marathon tasks to define custom service UI and HTTP endpoints, which are made available via `<dcos-cluster>/service/<service-name>`. This can be achieved by setting the following Marathon task labels:
-
+## Service Endpoint
+Admin Router offers `/service` endpoint which enables users to access easily
+some of the task launched on DC/OS. AR internally pre-fetches data from Mesos
```

Comment:
❓ Personally I'd like to `%s/AR/Admin Router` but this isn't new so this is optional.